process/accounts/altered_accounts_processor.go

```diff
@@ -0,0 +1,48 @@
+package accounts
+
+import (
+	"github.com/ElrondNetwork/covalent-indexer-go/api"
+	"github.com/ElrondNetwork/covalent-indexer-go/process/utility"
+	"github.com/ElrondNetwork/covalent-indexer-go/schemaV2"
+)
+
+type alteredAccountsProcessor struct {
+}
+
+// NewAlteredAccountsProcessor creates a new instance of altered accounts processor
+func NewAlteredAccountsProcessor() *alteredAccountsProcessor {
+	return &alteredAccountsProcessor{}
```

Comment:
this is not being referenced anywhere at least yet, will this be used in a future PR?

Reply:
Yes, it's not referenced because this feature does not even exist on mainnet yet.
Should be used in the future